Sources/Datadog/RUMMonitor.swift

```diff
@@ -597,19 +596,15 @@ public class RUMMonitor: DDRUMMonitor, RUMCommandSubscriber {
     // MARK: - RUMCommandSubscriber
 
     func process(command: RUMCommand) {
-        guard let scope = core.v1.scope(for: RUMFeature.self) else {
-            return
-        }
-
-        queue.async {
-            let transformedCommand = self.transform(command: command)
+        core.v1.scope(for: RUMFeature.self)?.eventWriteContext { context, writer in
+            self.queue.sync {
```

Comment:
All 3 Features now perform `sync` on the event write context. This is not set in stone, and I'm exploring ways of synchronising Features with the shared context.

Reply:
👍 Just submitted my review on that 🙂. Definitely, we need to explore the impact of this in high load situations. I'll think of possible solutions as well.